cfgov/unprocessed/css/molecules/overlay-hero.less

```diff
@@ -0,0 +1,134 @@
+
+/* topdoc
+  name: Theme variables
+  family: cfgov-organisms
+  notes:
+    - "The following sizing variables are exposed,
+       allowing you to easily override them before compiling."
+  patterns:
+  - name: Sizes
+    codenotes:
+      - |
+        @overlay-hero_sm
+        @overlay-hero_lg
+  tags:
+  - less
+*/
+@overlay-hero_sm: 100%;
+@overlay-hero_lg: 1230px;
+
+/* topdoc
+  name: Overlay Hero
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-overlay-hero" style="background-image: url(lg_img.jpg);">
+            <div class="content-l">
+                <section class="m-overlay-hero_content">
+                    <h1 class="m-overlay-hero_heading">Overlay Hero</h1>
+                    <p class="m-overlay-hero_text">
+                        Overlay Hero content
+                    </p>
+                </section>
+            </div>
+            <div class="m-overlay-hero_sm-img"
+                 style="background-image: url(sm_img.jpg);"></div>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-overlay-hero
+            .content-l
+              .m-overlay-hero_content
+                .m-overlay-hero_heading
+                .m-overlay-hero_text
+              .m-overlay-hero_sm-img
+
+  tags:
+    - cfgov-molecules
+*/
+
+.m-overlay-hero {
+    position: relative;
+
+    padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
+    padding-right: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+    padding-left: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
```

Comment:
Nevermind. I thought all three were set to the same value, but seeing now that right and left are half gutters.